week0002/daudau/php/daudau.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/*
+	explaination of my code. i got it from myself.
+	Do đề bài là theo hình xoắn ốc, ta nhận thấy nó đưọc chia thành các lớp, mỗi lớp là 1 "vòng tròn". đường thẳng cũng là vòng tròn :D
+	-> ý tưởng là dựa vào tọa độ của điểm trong array, ta cần xác định nó thuộc lớp thứ mấy của hình xoắn ốc và xác định giá trị tại điểm đó.
+	Example data:
+	N = 3; M = 4;
+	-------------
+	1	2	3	4
+
+	10	11	12	5
+
+	9	8	7	6
+	-------------
+*/
+
+//nhập giá trị từ user.
+fwrite(STDOUT, 'Input N: ');
```

Comment:
You may consider phrasing it clearer: `Input the number of rows N:`, `Input the number of columns M:`